effectsize_example.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Effect size"
+output: html_notebook
+---
+```{r}
+library(tidyverse)
+```
+
+
+## Data
+Assuming between-subjects design:
+```{r}
+set.seed(12)
+n <- 20
+data <- tibble(
+  Group = rep(c("A", "B"), each = n),
```

Comment:
Personal preference. Rationale: to distinguish column names from variable names, e.g., in ggplot().

Happy to change once we agree on one style guide.